CalPatRec/src/CalTimePeakFinder_module.cc

```diff
@@ -200,7 +200,7 @@ namespace mu2e {
     static const double twopi(2*pi);
 
     double              mphi(-9999.);
-    double              meanDriftTime = 1.25/0.06;// half straw tube radius / drift velocity
+    double              meanDriftTime = 46.7;//1.25/0.06;// half straw tube radius / drift velocity
```

Comment:
DriftSpeed (and hence meanDriftTime) should come from StrawResponse.

Reply:
removed old comment. The current number is a constant extracted from a diagnostic plot. This number differs from the Offline because of changes introduced in the Mu2eII branch